entrypoint.sh

```diff
@@ -32,8 +32,14 @@ apt -y install \
   lcov \
   lsb-release \
   python3-pip \
+  python3-vcstool \
   wget
 
+# Add colcon repository
+mkdir -p /etc/apt/keyrings/
```

Comment:
Alternatively, we can just add the ROS repository and install both colcon and vcstool from there.

Reply:
That is in fact another option, there are different ROS repositories that can serve us but I would prefer the lower scope of packages possible and the lower number of repos/configurations.

Reply:
Sounds good.